ClassLine/LineUnitTests/UnitTest1.cs

```diff
@@ -0,0 +1,81 @@
+using ClassLine;
+
+namespace LineUnitTests
+{
+    [TestFixture]
+    public class LineTests
+    {
+        [Test]
+        public void ConstructorTest()
+        {
+            var line = new Line(2.5, -1.3);
+            Assert.That(line.A, Is.EqualTo(2.5));
+            Assert.That(line.B, Is.EqualTo(-1.3));
+        }
+
+        [TestCase(double.NaN, 1)]
+        [TestCase(1, double.PositiveInfinity)]
+        public void Constructor_InvalidValues_Throws(double a, double b)
+        {
+            Assert.That(() => new Line(a, b), Throws.ArgumentException);
+        }
+
+        [TestCase(1, 0, 45)]
+        [TestCase(0, 0, 0)]
+        [TestCase(-1, 0, -45)]
+        public void AngleInDegreesTest(double a, double b, double expected)
+        {
+            var line = new Line(a, b);
+            Assert.That(line.AngleInDegrees, Is.EqualTo(expected).Within(1e-13));
+        }
+
+        [Test]
+        public void ToStringTest_PositiveAndNegativeB()
+        {
+            var line1 = new Line(1.5, -2.2);
+            var line2 = new Line(-3.0, 0.5);
+            Assert.That(line1.ToString(), Is.EqualTo("y = 1.5x - 2.2"));
+            Assert.That(line2.ToString(), Is.EqualTo("y = -3x + 0.5"));
+        }
```

Comment:
Лучше было сделать параметризованные тесты
```suggestion
        [TestCase(1.5, -2.2, "y = 1.5x - 2.2")]
        [TestCase(-3.0, 0.5, "y = -3x + 0.5")]
        public void ToStringTest_PositiveAndNegativeB(double k, double b, string result)
        {
            var line = new Line(k, b);
            Assert.That(line.ToString(), Is.EqualTo(result));
        }
```